climpred/prediction.py

```diff
@@ -16,15 +16,18 @@
     - PM_sig(ds, control, metric=rmse, comparison=_m2m, bootstrap=500, sig=99)
     - threshold to determine predictability horizon
 
+# TODO: make metrics non-dependent of prediction framework used
```

Comment:
This is definitely important to address eventually, but I am happy for now with RMSE and ACC for my simulations.

Reply:
mse also works like that now if you install my version of xskillscore